scripts/test-sim-backwards.sh

```diff
@@ -48,7 +48,10 @@ PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network --p
 PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network test bash "$SCRIPTDIR"/test-backwards.sh upload
 
 # this replaces anywhere that has "/release/" in the config file, which currently just renames the static dir paths
-sed -i -e 's#/release/#/branch/#g' `storj-sim network env SATELLITE_0_DIR`/config.yaml
+sed -i -e 's#/release/#/branch/#g' `storj-sim network env SATELLITE_API_0_DIR`/config.yaml
+
+# replace any 140XX port with 100XX port to fix, satellite.API part removal from satellite.Peer
+sed -i -e "s#$STORJ_NETWORK_HOST4:100#$STORJ_NETWORK_HOST4:140#g" `storj-sim network env SATELLITE_API_0_DIR`/config.yaml
```

Comment:
Don't we also need to update gateway? Or didn't we update it to use gateway?

Reply:
im not sure, let me check

Reply:
Also check whether the current release is using that, otherwise it'll break when we create a release after this.

Reply:
oh actually i think it was the integration test that used gateway, but the backwards compatibility tests don't

Reply:
the uplink currently uses gateway config to connect to things

Reply:
Verify that the `storj-sim` from current release doesn't configure gateway config with a `140XX` port. If it does, then we also need a fix for that for forward compatibility.

Reply:
```
~/go/src/storj.io/storj - ((HEAD detached at v0.24.3)) $ cat /Users/Jessica/Library/Application\ Support/Storj/Local-Network/gateway/0/config.yaml | grep -i :1
debug.addr: 127.0.0.1:11009
# satellite-addr: 127.0.0.1:10000
server.address: 127.0.0.1:11000
```

Reply:
I checked out the current release, did `go install`, and ran `storj-sim network setup` and looked at gateway config:
```
~/go/src/storj.io/storj - ((HEAD detached at v0.24.3)) $ cat /Users/Jessica/Library/Application\ Support/Storj/Local-Network/gateway/0/config.yaml | grep -i :1
debug.addr: 127.0.0.1:11009
server.address: 127.0.0.1:11000
```